app/models/github/issue_label.rb

```diff
@@ -3,4 +3,20 @@ class Github::IssueLabel < ApplicationRecord
     inverse_of: :labels,
     foreign_key: "github_issue_id",
     class_name: "Github::Issue"
+
+  %w[action knowledge module size status type].each do |label|
+    define_singleton_method "for_#{label}" do |val|
+      return unless const_get(label.pluralize.upcase.to_sym).include?(val)
+
+      "x:#{label}/#{val.to_s.tr('_', '-')}"
```

Comment:
The `_` to `-` translation is due to symbols not supporting dashes.

Reply:
Symbols do support dashes but you have to use a special syntax: `:"foo-bar"`

Where are the symbols here?